src/App.js

```diff
@@ -10,7 +11,8 @@ import {FirstReactComponent} from './FirstReactComponent.jsx';
       <FirstReactComponent />
       <ChildrenWithTitle title="I am title">
         I am children
-      </ChildrenWithTitle>
+      </ ChildrenWithTitle>
+      <ConditionalRenderingComponent/>
```

Comment:
AC - 2F is not met. We need to have two `ConditionalRenderingComponent` components